dotcom-rendering/src/model/enhanceCollections.ts

```diff
@@ -45,6 +45,7 @@ export const enhanceCollections = (
 			config: {
 				showDateHeader: collection.config.showDateHeader,
 			},
+			hasMore,
```

Comment:
I think additionally we'll want to check if the container config has hideShowMore enabled, and hide show more if it is!

![image](https://user-images.githubusercontent.com/21217225/201381668-ab949dcc-42fb-4754-b152-9501f4562418.png)

Reply:
ooh, good call! For some reason I thought that Frontend was looking at `hideShowMore` when it was deciding whether to set `hasMore`, but I guess not!

Reply:
So did I! I only noticed whilst reviewing #6392 !